.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+# Dependabot configuration.
+#
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Make it 9am UTC (on Mondays) by default.

```suggestion
      interval: "weekly"
      time: "09:00"
```